.github/workflows/pr_checker.yml

```
@@ -0,0 +1,56 @@
+name: Spoony pr checker
+on:
+  pull_request:
+    branches: [ develop, main ]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: .
+
+jobs:
+  build:
+    name: PR Checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Gradle cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: set up JDK 18
+        uses: actions/setup-java@v3
+        with:
+          java-version: 18
+          distribution: 'temurin'
+
+      - name: Change gradlew permissions
+        run: chmod +x ./gradlew
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Touch local properties
+        run: touch local.properties
+
+      - name: Access Local Properties
+        env:
+          HFM_BASE_URL: ${{ secrets.BASE_URL }}
+          TEST_BASE_URL: ${{ secrets.TEST_BASE_URL }}
+        run: |
+          echo base.url=\"$BASE_URL\" >> local.properties
+          echo test.base.url=\"$TEST_BASE_URL\" >> local.properties
```

P2: 파일에 URL을 설정하는 부분에서 환경변수를 잘못 참조하고 있는걸로 보여요
`env:` 에서 `HFM_BASE_URL`로 환경변수를 설정했는데, echo 명령어에서는 `$BASE_URL`로 사용하고 있네요. 
```suggestion
      - name: Access Local Properties
        env:
          HFM_BASE_URL: ${{ secrets.BASE_URL }}
          TEST_BASE_URL: ${{ secrets.TEST_BASE_URL }}
        run: |
          echo base.url=\"$HFM_BASE_URL\" >> local.properties
          echo test.base.url=\"$TEST_BASE_URL\" >> local.properties
```
이렇게 바꾸면 오류 없이 적용 될 것 같습니다!